### PR TITLE
Feat/buttons UI

### PR DESCRIPTION
**Pull request by johnkuney**
General updates of buttons page ui
- style updates
- modal for button creation
- transaction table

### CODE REVIEW

**Review by Klakurka — APPROVED** [review]

**Review by Klakurka — CHANGES_REQUESTED** [review]
Let's use commas on the amounts.

**Comment by Klakurka** [comment]
ID and time stamp can be updated to something better at some point.

**Comment by johnkuney** [comment]
Okay added

**Review by Klakurka — CHANGES_REQUESTED** [review]
Still an issue for me:

image

Should format the numbers so that 2 decimals are always showing and also using commas.

**Comment by johnkuney** [comment]
hmm okay. Are you seeing these transactions on the dev account? Cause I actually dont see any, so was using some mock data instead but must be strings instead of numbers or something. Will get it sorted though. 

Another thought, should I make the transaction ids link to that transaction on the block explorer (and for bch ones maybe blockchair?)

**Comment by Klakurka** [comment]
Maybe we can get a 2nd reviewer to confirm what they see.

For the transactions, let's use the tx hashes instead of an "ID" as I'm not sure what that is.

**Comment by johnkuney** [comment]
Okay was able to get the transactions working. Yeah my bad, hash makes more sense. Added number formatting, ticker based on address, and links to explorers

File: components/Paybutton/paybutton.module.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+}
+
+.paybutton_card,
+.paybutton_nocard {
+  background-color: var(--secondary-bg-color);
+  border-radius: 10px;
+  padding: 20px;
+  margin-bottom: 15px;
+  border: 1px solid var(--secondary-bg-color);
+  transition: all ease-in-out 200ms;
```
Comment (chedieck) [inline components/Paybutton/paybutton.module.css:15]: I find this `200ms` too slow

**Review by chedieck — CHANGES_REQUESTED** [review]
LGTM, I just left a remark for the animation transition that I found slow, don't know if others agree

**Review by Klakurka — CHANGES_REQUESTED** [review]
- time stamp should be a date, probably use the the user's local date format or whatever we're using on the site already if we are.
- would be nice if the hash column extended out to show the entire thing without the ellipsis if there's room (on larger screens) so people can more easily copy+paste them. OR add a copy button.

**Comment by johnkuney** [comment]
For sure, can add those 
And sure animation could be a bit quicker, changed timing 100ms

**Comment by johnkuney** [comment]
okay added, times should be local, using moment.js

**Review by Klakurka — APPROVED** [review]

**Pull request merged**